product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -35,14 +78,14 @@ Assume that your cluster is on an account called `development` and is being acce
     -   VPC: `vpc-client`
     -   VPC subnet: `snet-client`
 
-### Prerequisites
+#### Prerequisites
 
 To walk through an example in your own environment, you need:
 
 - Your cluster URL. You can find the URL in the **Connect** tab of your cluster instance in the BigAnimal portal.
 
 
-### Step 1: Create an endpoint service for your cluster
+#### Step 1: Create an endpoint service for your cluster
```

Comment:
The format of this Step is certainly acceptable.  I will add, however, that this is one area where screenshots become extremely useful for the customer.  May want to consider including some (as an exception to our rule) in the near future.  As a reference, see https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/02_connecting_your_cluster/01_connecting_from_azure/#step-1-create-an-azure-private-endpoint